server/src/main/java/org/opensearch/search/backpressure/stats/SearchBackpressureStats.java

```diff
@@ -24,40 +26,61 @@
 public class SearchBackpressureStats implements ToXContentFragment, Writeable {
     private final SearchShardTaskStats searchShardTaskStats;
     private final SearchBackpressureMode mode;
+    @Nullable
+    private final SearchTaskStats searchTaskStats;
 
-    public SearchBackpressureStats(SearchShardTaskStats searchShardTaskStats, SearchBackpressureMode mode) {
+    public SearchBackpressureStats(
+        SearchTaskStats searchTaskStats,
+        SearchShardTaskStats searchShardTaskStats,
+        SearchBackpressureMode mode
+    ) {
         this.searchShardTaskStats = searchShardTaskStats;
         this.mode = mode;
+        this.searchTaskStats = searchTaskStats;
     }
 
     public SearchBackpressureStats(StreamInput in) throws IOException {
-        this(new SearchShardTaskStats(in), SearchBackpressureMode.fromName(in.readString()));
+        searchShardTaskStats = new SearchShardTaskStats(in);
+        mode = SearchBackpressureMode.fromName(in.readString());
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            searchTaskStats = in.readOptionalWriteable(SearchTaskStats::new);
+        } else {
+            searchTaskStats = null;
+        }
     }
```

Comment:
Set version to V_2_6 and then backport

Reply:
I wish we had a better way to do this, but I agree that @reta's suggestion is the least disruptive (and avoids the bad look of merging a PR with failing tests).

Reply:
Wouldn't the BWC tests fail between the backport and the second main fix. The second problem is the automatic back port starts to fail since it would need a change to point to 2_6. Seems just too much overhead on backports. 
Instead if we can auto-trigger a backport PR and keep it handy before we merge to main it minimises the changes of build failures and simplifies the process.

@reta @andrross Thoughts?

Reply:
Thanks @Bukhtawar, yes, once 2.x backport is merged, the `main` would start to fail but:
 - the time window between `main` being broken and `2.x` being merged is unpredictable, could take hours blocking every other merge to `main`
 - the pull request to `main` could be sent right with backport to `2.x` - yes, it will fail, but it is just a matter of restarting the Gradle check to get it merged (after `2.x` merge)

The process is fair from ideal, but the disruption time would be minimal in this case, I think.